stylesheets/_component.datatables.scss

```diff
@@ -46,17 +46,28 @@
 }
 // scss-lint:enable SelectorFormat
 
-.dt-buttons {
-    border-right: 1px solid #ccc;
+a.dt-button {
+    border-right: 1px solid color(border);
+    margin-right: $margin-base-vertical;
+}
+
+.dt-buttons > a,
+.dt-buttons > button {
     float: left;
-    margin-top: 11px;
-    margin-right: 10px;
-    padding-right: 6px;
+    margin-top: $margin-large-vertical;
+    padding: 0 $padding-base-horizontal;
+}
+
+.dt-buttons > button {
+    margin-right: $margin-base-vertical;
 }
 
-.dt-button {
+button.dt-button {
```

Comment:
Could this be merged with `dt-buttons > button` on line 61? I'm not 100% sure but it looks like they both refer to the same thing, a button with the `.dt-button` class. Perhaps using the `dt-buttons > button.dt-button`?